packages/dd-trace/src/config.js

```diff
@@ -564,7 +564,15 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
           process.env.DD_GIT_PROPERTIES_FILE,
           `${process.cwd()}/git.properties`
         )
-        const gitPropertiesString = maybeFile(DD_GIT_PROPERTIES_FILE)
+        let gitPropertiesString
+        try {
+          gitPropertiesString = fs.readFileSync(DD_GIT_PROPERTIES_FILE, 'utf8')
+        } catch (e) {
+          // Only log error if the user has set a git.properties path
+          if (process.env.DD_GIT_PROPERTIES_FILE) {
+            log.error(e)
+          }
+        }
```

Comment:
```suggestion
        let gitPropertiesString
        if (process.env.DD_GIT_PROPERTIES_FILE) {
          gitPropertiesString = maybeFile(DD_GIT_PROPERTIES_FILE);
        }
```
How about this? It'd still delegate to maybeFile so we aren't duplicating much of its code here. You could even preserve this being `const` if you replaced the `if` with `?:` but that's secondary (so sad `if` isn't an expression but a statement…)